app/assets/stylesheets/application.scss

```diff
@@ -14,3 +14,6 @@
 @import "stafftools";
 
 @import "remodal/dist/remodal";
+
+// autocomplete
```

Comment:
The `@import` is pretty self explanatory here, would mind removing this comment?